views/layouts/layout.njk

```diff
@@ -1,20 +1,31 @@
 <!doctype html>
-<html>
+<html class="no-js">
   <head>
     <meta charset="utf-8">
     <meta name="robots" content="noindex, nofollow">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{{ title }}</title>
     <link href="/stylesheets/main.css" rel="stylesheet" media="all" />
-    <!--[if lt IE 9]><script src="/javascripts/ie.js"></script><![endif]-->
-    <script src="/javascripts/application.js"></script>
+    <!--[if lt IE 9]>
+      <script src="/javascripts/vendor/jquery-1.12.0.js"></script>
+      <script src="/javascripts/ie.js"></script>
+    <![endif]-->
+    <!--[if gte IE 9]><!-->
+      <script src="/javascripts/vendor/jquery-3.2.1.js"></script>
```

Comment:
do we need two version of jquery?

Reply:
I've not seen this done before, but my gut is that as long as we need to support IE8 we should just use the latest 1.x version, unless there's real benefits to doing it?

Reply:
Using 1.12 as a fallback for legacy IE is [recommended](https://jquery.com/browser-support/). However as I mentioned in the description, #83 also adds rebasing so this will need to be rebased anyway.

Reply:
I take your point @36degrees - we're only loading 1.12 now.